kahuna/public/js/util/queue.js

```diff
@@ -81,5 +89,6 @@ export const createQueue = ({
     }
   };
 
-  return { add };
+  const x = { add };
```

Comment:
Was this added for debugging purposes? It looks like it can be removed.

Reply:
Need to double check the JSDOC inference works without it.